inst/stan/estimate_infections.stan

```diff
@@ -156,6 +156,13 @@ transformed parameters {
   } else {
     obs_reports = reports[1:ot];
   }
+
+  // accumulate reports
+  if (any_accumulate) {
```

Comment:
Just checking the difference between estimate_infections and truncation is intentional (i.e one accumulates after truncation and the other before)